lib/minitest/failure_reporter_reporter.rb

```diff
@@ -42,6 +43,11 @@ def format(test)
 
       private
 
+      def normalize_file_path(file_path)
```

Comment:
Should this be in a different lib file? Since we will normalize the path often?

Reply:
Is this something we do often?  I thought this is quite a specific scenario.  @doodzik?

Reply:
We only do it in this specific instance so I think we can probably keep it as a private method. If we need to do it in another instance we can extract it.

Reply:
You could potentially monkey patch pathname with [a refinement](https://dzone.com/articles/how-not-to-cut-your-source-with-sharp-knife-as-mon) (so we aren't leaking the patch in other places).

Reply:
I'm going to merge as-is :)

Reply:
> You could potentially monkey patch Pathname with a refinement (so we aren't leaking the patch in other places). But that is probably overcomplicating it.

Please no monkey patch / refinement for this. I think this is fine as it is. If we need to reuse it I would prefer to extract a class.